dassana/common.py

```diff
@@ -32,6 +33,26 @@ def compress_file(file_name):
     logger.info("Compressed file completed")
 
 
+def delete_file(file_name):
+    os.remove(file_name)     
```

Comment:
Let's use [pathlib](https://docs.python.org/3/library/pathlib.html#pathlib.Path.unlink)
```suggestion
    Path.unlink(missing_ok=True)
```
That way, we don't need to check if the file exists or not before removing.